src/odb/src/codeGenerator/schema/chip/dbChip.json

```diff
@@ -0,0 +1,159 @@
+{
+  "name": "dbChip",
+  "type": "dbObject",
+  "ostream_scope": true,
+  "hash" : "0x1",
+  "enums": [
+    {
+      "name": "ChipType",
+      "class": true,
+      "values": [
+        "DIE",
+        "RDL", 
+        "IP",
+        "SUBSTRATE",
+        "HIER"
+      ],
+      "public": true
+    }
+  ],
+  "fields": [
+    {
+      "name": "name_",
+      "type": "std::string",
+      "schema": "db_schema_chip_extended",
+      "flags": ["no-set"]
+    },
+    {
+      "name": "type_",
+      "type": "uint",
+      "default": 0,
+      "schema": "db_schema_chip_extended",
+      "flags": ["private"]
+    },
+    {
+      "name": "offset_",
+      "type": "Point",
+      "default": "{}",
+      "schema": "db_schema_chip_extended"
+    },
+    {
+      "name": "width_",
+      "type": "int",
+      "default": "0",
+      "schema": "db_schema_chip_extended"
+    },
+    {
+      "name": "height_",
+      "type": "int", 
+      "default": "0",
+      "schema": "db_schema_chip_extended"
+    },
+    {
+      "name": "thickness_",
+      "type": "int",
+      "default": "0",
+      "schema": "db_schema_chip_extended"
+    },
+    {
+      "name": "shrink_",
+      "type": "float",
+      "default": "0.0",
+      "comment": "// range ]0, 1]",
```

Comment:
`[0, 1]` ?

Reply:
See https://en.wikipedia.org/wiki/Interval_(mathematics) "Notations for intervals"

Reply:
I'm ok to accept a PR changing it

Reply:
Sure #8018